tests/h/models/user_test.py

```diff
@@ -172,6 +172,12 @@ def test_User_activate_activates_user(self, db_session):  # noqa: N802
 
         assert user.is_activated
 
+    def test_instance_has_privacy_accepted_proptery(self, db_session):
+        user = models.User()
+
+        assert hasattr(user, 'privacy_accepted')
+        assert user.privacy_accepted is None  # nullable
```

Comment:
This can be done in a single assert using `getattr` (which would raise `AttributeError` if the attr doesn't exist, as opposed to existing with the value `None`).

Reply:
(Actually the whole test could be `assert getattr(models.User(), "privacy_accepted") is None`)